### ISSUE TITLE
Update Graduate (Eduardo Tunni)

### ISSUE THREAD

**Issue by thlinard** [google/fonts#4134]
Same as #4119

Version here & in the API: version 1.001, Western Europe languages support only.

Version [upstream](https://github.com/etunni/graduate): version 1.001, 8 years old. BUT: with [Adobe Latin 3](https://adobe-type-tools.github.io/adobe-latin-charsets/adobe-latin-3.html) support.

**Comment by thlinard** [comment]
Another possibility:  [Graduate VF](https://github.com/etunni/Graduate-Variable-Font).

**Comment by chrissimpkins** [comment]
@emmamarichal Is the upstream is working on it label still accurate here?  Are we waiting on anything to onboard this family?

**Comment by chrissimpkins** [comment]
And what is the needs manager's opinion question on this project?

**Comment by emmamarichal** [comment]
@chrissimpkins 
It's a bit like the Pathway project. We can upgrade the current Graduate (I can do that quickly), but there is also a variable version (quite complex) that we can add to the catalog.

I'm going to make a new issue to keep the two projects separate. Here is the email Eduardo Tunni wrote me about the variable: 

> Repect to Graduate is a little monster with 14 axes, and the source is in FontLab because at this time —2019— Glyhs only admitted 3 axes.
> This work was an experimentation of limits for variable fonts and it has axes that I don't think are ever used.
> The risk of migrating FontLab work to Glyphs is big.
> I can check in with Dave to see what thinks about this.

**Comment by emmamarichal** [comment]
Variable version issue : https://github.com/google/fonts/issues/6108

**Comment by chrissimpkins** [comment]
@vv-monsalve It looks like the VF format update in this project will require new axis registrations.
